twifbeddit-app/backend/src/mongo.js

```diff
@@ -36,7 +37,8 @@ var PostSchema = new mongoose.Schema({
     anonymous: {type: Boolean, default: false},
     topic: {type: String, default: null},
     post_type: {type: String, default: "text"},
-    text: {type: String, required: [true, "cannot be empty."]}
+    text: {type: String, required: [true, "cannot be empty."]},
```

Comment:
Actually I shouldn't have made this required, can we remove that?

Reply:
Not sure, we could have the text hold an webUrl.